.gitignore

```diff
@@ -1,2 +1,16 @@
-target
+### Created by https://www.gitignore.io
+### Rust ###
+# Generated by Cargo
+# will have compiled files and executables
+debug/
+target/
+
+# Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
+# More information here https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html
 Cargo.lock
+
+# These are backup files generated by rustfmt
+**/*.rs.bk
+
+# MSVC Windows builds of rustc generate these, which store debugging information
+*.pdb
```

Comment:
How come you are changing the `.gitignore`?

Reply:
Force of habit, it carried over from my previous work on `roff-rs`, I can revert this if you want to.

Reply:
Please do so.  imo person/machine specific ignores should be set in the personal ignores rather than checked into the shared ignores.